okhttp-tests/src/test/java/okhttp3/internal/platform/Jdk9PlatformTest.java

```diff
@@ -40,7 +41,10 @@ public void findsAlpnMethods() {
   }
 
   @Test
-  public void testToStringIsClassname() {
-    assertThat(new Jdk9Platform(null, null).toString()).isEqualTo("Jdk9Platform");
+  public void testToStringIsClassname() throws NoSuchMethodException {
+    assumeTrue(getPlatform().equals("jdk9"));
```

Comment:
I was getting a strange NPE locally, but seemed valid given I was faking data.  Not sure we should assume this runs on other platforms.